src/target/rp.c

```diff
@@ -615,19 +638,40 @@ static void rp_spi_read(
 	target_mem_write32(t, RP_SSI_ENABLE, ssi_enabled);
 }
 
-#if 0
+/* Checks if the QSPI and XIP controllers are in their POR state */
+static bool rp_flash_in_por_state(target_s *const t)
+{
+	const uint32_t clkEnables0 = target_mem_read32(t, RP_CLOCKS_WAKE_EN0);
+	const uint32_t clkEnables1 = target_mem_read32(t, RP_CLOCKS_WAKE_EN1);
```

Comment:
Camel case variables?

Reply:
good catch - we'll amend this particular commit in the patch set